### DISCUSSION TITLE
Can I disable autoplay? It keeps wasting bandwith when I only want to reply to comments.

### DISCUSSION THREAD

**Discussion by mzso** [ReVanced/revanced-discussions#1056] [Question]
Hi! I would like to prevent autoplay, but I can't seem to find how. Maybe I'm missing something, but the video always starts playing when I open youtube RV from push or e-mail message.

Also, it seems I can't reply if I open the app via e-mail message. I always get a nondescriptive "an error occured" message.

**Comment by LisoUseInAIKyrios** [comment]
After opening a video link it will start prebuffering even if video playback did not start, so the bandwidth used is the same.

I cannot reproduce any issues opening links from email or push notification.  Make sure you are on the latest patch release and if it still acts up try turning off iOS spoofing under `Misc -> Spoof client`.  Unless you want to keep that error because it will prevent using video bandwidth.

**Reply by mzso** [reply]
@LisoUseInAIKyrios 

Okay, so this "error occured"  when replying keeps happening all the time still.
But one thing I realized is that when I open the replies from push alerts or or withing the app notifications, it takes me to comments without the video even being loaded, but when opened from the e-mail notifications I get the video, with the comment in question under it. (and also the error) _So maybe Revanced could load the same thing for url-s opened from gmail as the other types of notifications?_

Also, a bandwidth saving feature could be to never autoplay/autobuffer videos, and only do so when someone taps play.

**Comment by mzso** [comment]
So, how about disabling pre-buffering? Not possible? (Am I the only one annoyed with morbidly overpriced mobile data?)
But auto play in itself is often annoying, especially when instantly playing something loud. The speaker volume tends to be set too loud without my doing. (I think it's VLC's fault, usually. I only use it with a headphone, but the loudspeaker volume also changes.)


I'll check out what's newest. It doesn't prevent using bandwidth, only successfully submitting the comment breaks. (I think this is only from e-mails.)
